examples/README.md

```diff
@@ -189,14 +189,143 @@ Unmaintained.  As a close alternative, see the jupyter notebook file
 
 ### Usage
 
-1. Compile malmo https://github.com/Microsoft/malmo, see
-   https://github.com/microsoft/malmo/blob/master/doc/build_linux.md
+1. Clone malmo https://github.com/Microsoft/malmo
+  Note: No need to compile malmo as a pre-built version (MalmoPython) has been included
+  under `../rocca/malmo`. Only install the required dependencies to launch minecraft.
 
-2. After compiling, copy
-   `<MALMO>/build/install/Python_Examples/MalmoPython.so` and
-   `<MALMO>/build/install/Python_Examples/malmoutils.py` under
-   `<ROCCA>/examples/malmo`.
+2. Launch `<MALMO>/Minecraft/launchClient.sh`.
 
-3. Launch `<MALMO>/Minecraft/launchClient.sh`.
+3. Launch chase malmo `python chase_malmo.py`.
 
-4. Launch chase malmo `python chase_malmo.py`.
+
+## Collect diamonds
+
+### Description
+
+A simple minecraft environment which includes blocks of diamonds locked 
+inside a house and a key to the house. The agent must look for 
+the key, open the house and collect a diamond to get a reward.
+
+### Status
+
+The Rocca agent is able to learn and plan the desired cognitive schematics 
+leading to getting/accumulating a reward via pattern mining and temporal deduction.
+
+The three optimal action plans required to get a reward are:
```

Comment:
These are the optimal mono-action plans.  Could you add as well the poly-action plans?  It's OK if it makes the `README.md` very large (I think I'm actually gonna split that examples folder into sub folders, one for each experiment, to minimize bloating).